File: app/src/main/java/tpcreative/co/qrscanner/model/TypeCategories.java
package tpcreative.co.qrscanner.model;

import java.io.Serializable;

public class TypeCategories implements Serializable{
    public int id;
    public String type;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }


    public TypeCategories(){
        this.id = 0;
        this.type = "";
    }

    public TypeCategories(int id, String type){
        this.id = id;
        this.type = type;
    }

}

File: app/src/main/java/tpcreative/co/qrscanner/model/TypeCategories.kt
package tpcreative.co.qrscanner.model
import java.io.Serializable

class TypeCategories : Serializable {
    var id: Int?
    var mType: String?

    fun getId(): Int {
        return id ?: 0
    }

    fun setId(id: Int) {
        this.id = id
    }

    fun getType(): String? {
        return mType
    }

    fun setType(type: String?) {
        this.mType = type
    }

    constructor() {
        id = 0
        mType = ""
    }

    constructor(id: Int, type: String?) {
        this.id = id
        this.mType = type
    }
}